instrumentation/jaxws/jaxws-2.0-cxf-3.0/javaagent/jaxws-2.0-cxf-3.0-javaagent.gradle

```diff
@@ -0,0 +1,34 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.apache.cxf"
+    module = "cxf-rt-frontend-jaxws"
+    versions = "[3.0.0,)"
+    assertInverse
+  }
+}
+
+dependencies {
+  implementation project(':instrumentation:jaxws:jaxws-2.0-cxf-3.0:library')
+
+  api group: 'javax.xml.ws', name: 'jaxws-api', version: '2.3.1'
```

Comment:
Won't using `api` include this lib in the javaagent? Consider using `compileOnly` + `testImplementation` instead

Reply:
fixed